templates/logrotate.rails

```diff
@@ -2,13 +2,18 @@
 /var/log/rails/*.log {
   su root root
   daily
-  size 50M
   rotate 3
   missingok
   notifempty
   delaycompress
-  copytruncate
+  sharedscripts
   postrotate
+    # Unicorn - reopen all logfiles
+    if [ {{ loggly.refresh_unicorn }} = True ]; then
+      master_pid=$(ps aux | grep '[u]nicorn master' | awk '{print $2}')
```

Comment:
So the brackets are for grep to not match against its own process: http://unix.stackexchange.com/questions/74185/how-can-i-prevent-grep-from-showing-up-in-ps-results